### ISSUE TITLE
KeepAspectRatio, sizes and orientation

### ISSUE THREAD

**Issue by carrbrpoa** [NativeScript/nativescript-camera#98]
Hello,

As and enhancement, is it possible that `width` and `height` could be treated conditionally when `KeepAspectRatio` is set to `true`, depending on camera orientation when photo is taken?

Motive: let's say I have the following options setted:

```
takePicture({
	width: 640,
	height: 480,
	keepAspectRatio: true,
...
```

As a result, I see that landscape/horizontal photo I take, is like 576x324 (_Ok_ case). 

Also see that portrait/vertical photo is 648x1152 (because of the "rule" that says that only one of width/height could be respected with `keepAspectRatio: true`), but actually, in this case, would be better for me to specify that when taking portrait photo, I would like to have maybe `width: 480` and `height: 640`.

**Comment by elena-p** [comment]
AC:
- [x] When keepAspectRatio is set to true, treat provided width and height according to screen orientation - in landscape use the bigger of the two values as width and in portrait use the bigger of the two values as height
- [x] When keepAspectRatio is set to false, provide image with the specified dimensions regardless of the screen orientation (current behaviour)
- [x] Demo and angular demo should be updated with options to enter keepAspectRatio, width and height; The demos should display width, height and scale of the taken image

**Comment by zbranzov** [comment]
The feature is released in nativescript-camera@4.1.0. Enjoy!

**Comment by carrbrpoa** [comment]
> The feature is released in nativescript-camera@4.1.0. Enjoy!

Actually since I suggested that, I needed to embed the camera with another plugin, because sometimes my app was being "chosen" to be killed in low resources device, when it was to switch to camera app. I know it's Android's behavior, so it was the solution I found.
